src/components/menu/editor-mode-controls.jsx

```diff
@@ -8,19 +9,24 @@ import NotebookTaskButton from "./notebook-task-button";
 
 import tasks from "../../actions/task-definitions";
 
+// the min-width of 300px keeps the title bar centered.
```

Comment:
where is this min-width? the thing below is empty. is this comment still relevant?

Reply:
yeah dead comment, will remove